packages/fluxible-router/package.json

```diff
@@ -2,13 +2,16 @@
   "name": "fluxible-router",
   "version": "2.0.0-beta.12.0",
   "description": "Routing for Fluxible applications",
-  "main": "index.js",
+  "main": "./dist/cjs/index.js",
```

Comment:
Do you need a module property for the es modules code now too?

Reply:
Yes! You are completely right!